doc/connectivity/bluetooth/api/gatt.rst

```diff
@@ -47,6 +47,10 @@ callback, a ``write`` callback and a set of permission. Both read and write
 callbacks can be set to NULL if the attribute permission don't allow their
 respective operations.
 
+.. note::
+   32-bit UUIDs are not supported in GATT. All 32-bit UUIDs shall be converted
+   to 128-bit UUIDs when the UUID is contained in an ATT PDU.
```

Comment:
Do we have a conversion function/macro for this? If not we should consider adding one

Reply:
There is [one ](https://github.com/zephyrproject-rtos/zephyr/blob/c2c467a6ffd2f22b12de82569494342663fa41ca/subsys/bluetooth/host/uuid.c#L31)but the only place its used is in a [UUID comparison function](https://github.com/zephyrproject-rtos/zephyr/blob/c2c467a6ffd2f22b12de82569494342663fa41ca/subsys/bluetooth/host/uuid.c#L50).

Reply:
Should be fairly straight forward to move that function to the public API since there's a clear usecase for application to use it here :) 

If you are up for it, you can do it, else I would suggest to create a Github enhancement issue

Reply:
I can take a look at it